### PR TITLE
W-14023536 filter by custom fields jh

### PR DESCRIPTION
**Pull request by JennyHajee**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/to-find-info.adoc [hunk 1]
```diff
@@ -68,15 +68,56 @@ Keyword or term that describes a specific search.
...
+|Data Type |Example Name for Data Type|Operator
+|Enum|Days of the week|Contains and not contains. The default value is contains.
+|Number|Quantity |Equals, not equals to, greater than, less than, greater than or equal to, and less than or equal to. The default value is equals.
+|Date |Release Date |Equals, not equals to, greater than, less than, greater than or equal to, and less than or equal to. The default value is equals.
+|Text|Team |Equals and not equals to. The default value is equals.
+|List of numbers|Orders |Contains and not contains. The default value is contains.
+|List of strings|Emails |Contains and not contains. The default value is contains.
+|===
+
+Use preset operators to filter the search results. The values of the operators can be changed and customized. For example, you can select a date from the list of the values or add a date by entering a custom date in the defined format (YYY-MM-DD). The customized value shows in the list of options as freeform.
```
Comment (guillerecalde) [inline modules/ROOT/pages/to-find-info.adoc]:
```suggestion
Use preset operators to filter the search results. The values of the operators can be changed and customized. For example, you can select a date from the list of the values or add a date by entering a custom date in the defined format (YYYY-MM-DD). The customized value shows in the list of options as freeform.
```

Comment (JennyHajee) [reply modules/ROOT/pages/to-find-info.adoc]: Thank you, fixed the typo.

File: modules/ROOT/pages/to-find-info.adoc [hunk 2]
```diff
@@ -68,15 +68,56 @@ Keyword or term that describes a specific search.
...
+. To find an option from a long list, use the scroll bar or enter a search term.
 . Click one or more options from the menu and click *Apply*. 
 . To clear a filter option, click *Clear filter*.
 . To clear all filter options, click *Clear all filters*.
 
+=== Filter By Custom Fields
+Custom fields are created by Exchange administrators and organization owners using the Exchange API to extend the Exchange asset metadata model and add custom data to any type of asset. The custom field filter lets you refine a search using preset information.
+
+
+The custom field filter has the following data types:
```
Comment (guillerecalde) [inline modules/ROOT/pages/to-find-info.adoc]: The custom field filter does not actually "have" a data type. What it does is show a set of operators depending on the custom field data type. For example, when the custom field data type is `Text`, the filter shows operators that are valid on text fields, such as "equal" or "not equal".

File: modules/ROOT/pages/to-find-info.adoc [hunk 3]
```diff
@@ -68,15 +68,56 @@ Keyword or term that describes a specific search.
...
+
+[%header,cols="1a,1a,2a"]
+|===
+|Data Type |Example Name for Data Type|Operator
```
Comment (guillerecalde) [inline modules/ROOT/pages/to-find-info.adoc]: In my opinion, the `Example name for data type` does not add useful information here

File: modules/ROOT/pages/to-find-info.adoc [hunk 4]
```diff
@@ -68,15 +68,56 @@ Keyword or term that describes a specific search.
...
+|===
+
+Use preset operators to filter the search results. The values of the operators can be changed and customized. For example, you can select a date from the list of the values or add a date by entering a custom date in the defined format (YYYY-MM-DD). The customized value shows in the list of options as freeform.
+
+//screen capture to show the custom field operator and value
+
+To filter a search using custom fields:
+
+. In the landing page, click *Any custom field*.
+. Use the scroll bar or enter a name for the data type to find an option in a long list.
```
Comment (guillerecalde) [inline modules/ROOT/pages/to-find-info.adoc]:
```suggestion
. Use the scroll bar or enter the name of the custom field to find an option in a long list. 
```

File: modules/ROOT/pages/to-find-info.adoc [hunk 5]
```diff
@@ -68,15 +68,56 @@ Keyword or term that describes a specific search.
...
 
-. From the Exchange landing page, select a filter option.
+. In the Exchange landing page, select a filter option.
+. To find an option from a long list, use the scroll bar or enter a search term.
 . Click one or more options from the menu and click *Apply*. 
 . To clear a filter option, click *Clear filter*.
 . To clear all filter options, click *Clear all filters*.
 
+=== Filter By Custom Fields
+Custom fields are created by Exchange administrators and organization owners using the Exchange API to extend the Exchange asset metadata model and add custom data to any type of asset. The custom field filter lets you refine a search using preset information.
```
Comment (nescalante) [inline modules/ROOT/pages/to-find-info.adoc]: I would add a link to [the custom fields page](https://docs.mulesoft.com/exchange/to-manage-custom-fields) here

File: modules/ROOT/pages/to-find-info.adoc [hunk 6]
```diff
@@ -68,15 +68,56 @@ Keyword or term that describes a specific search.
...
+
+
+Custom fields use the following data types:
+
+[%header,cols="1a,2a"]
+|===
+|Data Type |Operator
+|Enum|Contains and not contains. The default value is contains.
```
Comment (nescalante) [inline modules/ROOT/pages/to-find-info.adoc]: the "default value" doesn't add value in this context IMO, user will have it visible anyway

**Review by nescalante — DISMISSED** [review]

File: modules/ROOT/pages/to-find-info.adoc [hunk 7]
```diff
@@ -68,15 +68,58 @@ Keyword or term that describes a specific search.
...
 
-Filter options appear when they are available within an organization. 
+* Custom field
++
+Custom fields are specific metadata fields set by the administrator of a business group or organization owner.
+
+Filter options appear when they are available within an organization.
 
-To filter search results:
+=== Filter By Type, Organization, Category, and Lifecycle State
```
Comment (hannanelson) [inline modules/ROOT/pages/to-find-info.adoc:77]:
```suggestion
=== Filter by Type, Organization, Category, and Lifecycle State
```

File: modules/ROOT/pages/to-find-info.adoc [hunk 8]
```diff
@@ -68,15 +68,58 @@ Keyword or term that describes a specific search.
...
 
-. From the Exchange landing page, select a filter option.
+. In the Exchange landing page, select a filter option.
+. To find an option from a long list, use the scroll bar or enter a search term.
 . Click one or more options from the menu and click *Apply*. 
 . To clear a filter option, click *Clear filter*.
 . To clear all filter options, click *Clear all filters*.
 
+=== Filter By Custom Fields
+Custom fields are created by Exchange administrators and organization owners using the Exchange API to extend the Exchange asset metadata model and add custom data to any type of asset. Assets can be filtered by custom fields to refine a search using preset information. The custom field filter is available only when assets in your organization have custom fields.
```
Comment (hannanelson) [inline modules/ROOT/pages/to-find-info.adoc:86]:
```suggestion
Exchange administrators and organization owners create custom fields using the Exchange API to extend the Exchange asset metadata model and add custom data to any type of asset. You can refine a search using preset information by filtering assets by custom fields. The custom field filter is available only when assets in your organization have custom fields.  
```

Comment (hannanelson) [reply modules/ROOT/pages/to-find-info.adoc:86]: Suggestions to make more active. The first sentence has a lot going on.

File: modules/ROOT/pages/to-find-info.adoc [hunk 9]
```diff
@@ -68,15 +68,58 @@ Keyword or term that describes a specific search.
...
+Custom fields are created by Exchange administrators and organization owners using the Exchange API to extend the Exchange asset metadata model and add custom data to any type of asset. Assets can be filtered by custom fields to refine a search using preset information. The custom field filter is available only when assets in your organization have custom fields.  
+
+For more information about creating custom fields for assets, see xref:to-manage-custom-fields.adoc[Manage Custom Fields].
+
+Custom fields use the following data types:
+
+[%header,cols="1a,2a"]
+|===
+|Data Type |Operator
+|Enum|Contains and not contains.
```
Comment (hannanelson) [inline modules/ROOT/pages/to-find-info.adoc]: If no cell is a full sentence, no periods for all.

File: modules/ROOT/pages/to-find-info.adoc [hunk 10]
```diff
@@ -68,15 +68,58 @@ Keyword or term that describes a specific search.
...
+|Data Type |Operator
+|Enum|Contains and not contains. 
+|Number|Equals, not equals to, greater than, less than, greater than or equal to, and less than or equal to. 
+|Date  |Equals, not equals to, greater than, less than, greater than or equal to, and less than or equal to. 
+|Text |Equals and not equals to. 
+|List of numbers |Contains and not contains. 
+|List of strings |Contains and not contains. 
+|===
+
+Use the preset operators to filter the search results. The values of the operators can be changed and customized. For example, you can select a date from the list of the values or add a date by entering a custom date in the defined format (YYYY-MM-DD). The customized value shows in the list of options as freeform. You can add as many custom fields as you need.
```
Comment (hannanelson) [inline modules/ROOT/pages/to-find-info.adoc:103]:
```suggestion
Use the preset operators to filter the search results. You can change and customize the values of the operators. For example, select a date from the list of values or add a date by entering a custom date in the defined format (YYYY-MM-DD). The customized value shows in the list of options as freeform. You can add as many custom fields as you need.
```

File: modules/ROOT/pages/to-find-info.adoc [hunk 11]
```diff
@@ -68,15 +68,58 @@ Keyword or term that describes a specific search.
...
+
+The following example shows the search filter with two custom fields:
+
+image:ex-custom-field-operator-value.png[20%,20%]
```
Comment (hannanelson) [inline modules/ROOT/pages/to-find-info.adoc]: You need alt text here.

File: modules/ROOT/pages/to-find-info.adoc [hunk 12]
```diff
@@ -68,15 +68,58 @@ Keyword or term that describes a specific search.
...
+
+To filter a search using custom fields:
+
+. In the landing page, click *Any custom field*.
+. Use the scroll bar or enter the name of the custom field to find an option in a long list. 
+. Select a custom field option from the list.
+. If you want to change the operator, select a operator from the drop-down list. 
+. Click *Select value(s)* and select a value for the operator from the list or enter a custom value. You can select more than one value.
+. If you want to add more custom fields to further filter the search, click *Add Custom Field* and select an option from the list.
```
Comment (hannanelson) [inline modules/ROOT/pages/to-find-info.adoc:116]:
```suggestion
. To add more custom fields to further filter the search, click *Add Custom Field* and select an option from the list.
```

Comment (hannanelson) [reply modules/ROOT/pages/to-find-info.adoc:116]: Suggest removing the preamble from the next few, too.

File: modules/ROOT/pages/to-find-info.adoc [hunk 13]
```diff
@@ -118,7 +161,5 @@ Administrators can save searches visible to anyone in their current business gro
 . Click *Save search*.
 . Confirm that your search appears in the saved searches in the left navigation bar.
 +
-. To rename a saved search, click the three dots and select *Rename*.
-. To delete a saved search, click the three dots and select *Delete*.
-
-
+. If you want to rename a saved search, click the three dots and select *Rename*.
```
Comment (hannanelson) [inline modules/ROOT/pages/to-find-info.adoc:164]:
```suggestion
. To rename a saved search, click the three dots and select *Rename*.
```

File: modules/ROOT/pages/to-find-info.adoc [hunk 14]
```diff
@@ -118,7 +161,5 @@ Administrators can save searches visible to anyone in their current business gro
...
+. If you want to delete a saved search, click the three dots and select *Delete*.
```
Comment (hannanelson) [inline modules/ROOT/pages/to-find-info.adoc:165]:
```suggestion
. To delete a saved search, click the three dots and select *Delete*.
```

**Review by hannanelson — DISMISSED** [review]

**Review by hannanelson — APPROVED** [review]

**Pull request merged**